components/chat_widget/src/components/ocs-chat/ocs-chat.css

```diff
@@ -662,7 +662,6 @@ textarea {
 }
 
 .message-bubble-assistant .chat-markdown {
-  @apply prose prose-sm prose-gray max-w-none prose-a:no-underline hover:prose-a:underline;
```

Comment:
This is already applied to the `.chat-markdown` class. Applying it again here messes things up.